package.json

```diff
@@ -11,16 +11,14 @@
     "LICENSE"
   ],
   "scripts": {
-    "lint": "eslint . --fix",
+    "fix": "eslint . --fix",
     "test": "jest",
-    "prehusky": "npm run lint",
```

Comment:
actually linting is automatically done before each commit with this command

Reply:
"lint" is now part of "test" which still runs on each commit ("husky")

Reply:
👍